### PR TITLE
update: update home page, fix sensor

### PR DESCRIPTION
**Pull request by GyonGyon**

### CODE REVIEW

File: app/javascript/src/pages/home/index.js [hunk 1]
```diff
@@ -30,14 +30,10 @@ const bindSearchBar = () => {
 }
 
 const bindTrackDapplist = (container) => {
-  container.each(function() {
+  container.find('.dapp').each(function() {
     const dapp = $(this)
-    const category = dapp.attr('data-category')
+    const category = dapp.attr('data-catagory')
```
Comment (classicalliu) [inline app/javascript/src/pages/home/index.js]: category, not catagory

File: app/javascript/src/pages/more/index.js [hunk 2]
```diff
@@ -8,21 +8,16 @@ import { titlebarNormal } from '../../utils/tablebar'
 import { trackDapplist } from '../../utils/sensors'
 
 const bindTrackDapplist = (container) => {
-  container.each(function() {
-    const dapp = j(this)
-    const category = dapp.attr('data-category')
+  container.find('.dapp').each(function () {
+    const dapp = $(this)
+    const category = dapp.attr('data-catagory')
```
Comment (classicalliu) [inline app/javascript/src/pages/more/index.js]: category

File: app/javascript/src/utils/neuron.js [hunk 3]
```diff
@@ -55,4 +52,64 @@ const initNeuronTable = () => {
...
+
+// trycatch way
+const warnPlatformError = () => console.warn('Not in Neuron')
+
+const openSearchPage = function() {
+  try {
+    window.appHybrid.startAddWebsitePage()
+  } catch (error) {
+    try {
+      window.touchSearchbar()
```
Comment (classicalliu) [inline app/javascript/src/utils/neuron.js]: Replace window.appHybrid && window.appHybrid.startAddWebsitePage()

Comment (GyonGyon) [reply app/javascript/src/utils/neuron.js]: Fixed

**Pull request merged**